virtual-server/examples/python/kubevirtclient.py

```diff
@@ -0,0 +1,269 @@
+import os
+import six
+
+from kubernetes import client, config, watch
+from kubernetes.client.exceptions import (
+    ApiTypeError,
+    ApiValueError
+)
+
+
+class KubeVirtClient:
+    """
+    Since native python client for kubevirt (https://github.com/kubevirt/client-python) has unresolved issues,
+    this class demonstrates basic operations on kubevirt.
+    """
+    def __init__(self, kubeconfig_path=None):
+        if kubeconfig_path is None:
+            kubeconfig_path = os.environ.get("KUBECONFIG")
+            if kubeconfig_path is None:
+                kubeconfig_path = os.path.expanduser("~/.kube/config")
+        config.kube_config.load_kube_config(kubeconfig_path)
+
+        self.api = client.CustomObjectsApi()
+        self.api_client = self.api.api_client
+
+    def kubevirt_api(self, namespace, name, command,
+                         group='subresources.kubevirt.io', version='v1', plural='virtualmachines',
+                         resource_path='/apis/{group}/{version}/namespaces/{namespace}/{plural}/{name}/{command}',
+                         method='PUT',
+                         **kwargs):
+        """
+        This method changes the state of the kubevirt instance. It is modified version of 
```

Comment:
Language change:
"This method changes the state of the kubevirt VirtualMachine instance"